modules/pipeline/providers/dbgc/dbgc.service.go

```diff
@@ -63,17 +73,16 @@ func (r *Reconciler) PipelineDatabaseGC(ctx context.Context) {
 	}
 }
 
-// query the data in the database according to req paging to perform gc
-func (r *Reconciler) doPipelineDatabaseGC(req apistructs.PipelinePageListRequest) {
+// doPipelineDatabaseGC query the data in the database according to req paging to perform gc
+func (r *dbgcService) doPipelineDatabaseGC(req apistructs.PipelinePageListRequest) {
 	var pageNum = req.PageNum
 
 	for {
 		req.PageNum = pageNum
-
 		pipelineResults, _, _, _, err := r.dbClient.PageListPipelines(req)
 		pageNum += 1
 		if err != nil {
-			logrus.Errorf("doPipelineDatabaseGC failed to compensate pipeline req %v err: %v", req, err)
+			logrus.Errorf("doPipelineDatabaseGC failed to compensate pipeline req: %v err: %v", req, err)
```

Comment:
add ref to provider, and use p.Log for logging.

Reply:
failed to xxx, req: %v, err: %v

Reply:
done